src/version/ContractVersionBase.sol

```diff
@@ -8,6 +8,6 @@ import {IVersionedContract} from "../interfaces/IVersionedContract.sol";
 contract ContractVersionBase is IVersionedContract {
     /// @notice The version of the contract
     function contractVersion() external pure override returns (string memory) {
-        return "1.4.0";
```

Comment:
we shouldn't be updating the versions separately from changesets, because then we don't get the release and documented changed versions in github.  how about adding a changeset, and then we do another release or version branch that updates the version via the changeset?

Reply:
yep reverted version bump